tmt/steps/finish/shell.py

```diff
@@ -60,4 +60,4 @@ def go(self, guest):
         # Execute each script on the guest
         for script in scripts:
             self.verbose('script', script, 'green')
-            guest.execute(script, cwd=self.step.workdir)
+            guest.execute(script, cwd=self.step.plan.run.workdir_tree)
```

Comment:
Do we want to share the `tree` directory across multiple plans? This could cause problems once/if we run plans in parallel.

Reply:
I guess it would be better not to share it ...